core/ir/riscv64/instr_create_api.h.in

```diff
@@ -138,25 +140,17 @@
  * \param m   The destination memory opnd.
  * \param r   The source register opnd.
  */
-#define XINST_CREATE_store_simd(dc, m, r) instr_create_1dst_1src(dc, OP_c_nop, m, r)
-
-/**
- * This platform-independent macro creates an instr_t for an indirect
- * jump through memory instruction.  For AArch32, the memory address
- * must be aligned to 4.
- * \param dc  The void * dcontext used to allocate memory for the instr_t.
- * \param m   The memory opnd holding the target.
- */
-#define XINST_CREATE_jump_mem(dc, m) \
```

Comment:
We don't want to define this for RISC-V? (and some others that are removed as well)

Reply:
Same as AArch64, there is no "load into PC" kind of instruction, so no `jump_mem`. All the removed macros are similar, they cannot be implemented on RISC-V.